qc_schema/dev/properties/calcinfo_properties.py

```diff
@@ -0,0 +1,35 @@
+"""
+The complete list of the general computation properties.
+"""
+
+calcinfo_properties = {}
+
+calcinfo_properties["calcinfo_nbasis"] = {
+    "type": "number",
+    "multipleOf": 1.0,
+    "description": "The number of basis functions for the computation."
+}
+
+calcinfo_properties["calcinfo_nmo"] = {
+    "type": "number",
+    "multipleOf": 1.0,
+    "description": "The number of molecular orbitals for the computation."
+}
+
+calcinfo_properties["calcinfo_nalapha"] = {
+    "type": "number",
+    "multipleOf": 1.0,
+    "description": "The number of alpha electrons in the computation."
+}
+
+calcinfo_properties["calcinfo_nbeta"] = {
+    "type": "number",
+    "multipleOf": 1.0,
+    "description": "The number of beta electrons in the computation."
+}
```

Comment:
Does something like reference (RHF/UHF/ROHF) go in `calcinfo_properties` or `scf_properties`? I guess I am not clear on the border where different steps in a computation get news instances of the schema. For example, you may run the SCF step with ROHF but then CCSD with UHF, and if both SCF/CCSD steps are part of the same instance, something (like the reference) needs to be present in multiple places.

This is related to the discussion about properties (HF vs. CCSD dipole moment); I think the same thing applies.

Reply:
I'm not sure there is a clear line yet. To me, I think thats a single computation. I kind of feel those specification might come in method as the choice to use ROHF SCF/UHF CCSD is a user setting. `properties` seems to be purely as the result.